Client/Frontend/Browser/Playlist/PlaylistViewController.swift

```diff
@@ -0,0 +1,1348 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import BraveShared
+import Shared
+import AVKit
+import AVFoundation
+import SDWebImage
+import CoreData
+import Data
+
+private let log = Logger.browserLogger
+
+// MARK: - DisplayMode
+
+private enum DisplayMode {
+    case iPhoneLayout
```

Comment:
Could we reuse `UIUserInterfaceIdiom` values maybe?

Reply:
I thought about that, but that has all sorts of other stuff as well like `unknown` and `tv` and `CarPlay`. I'm not sure if it's better to keep this enum or have a boolean or use the `UIUserInterfaceIdiom`.

@soner-yuksel @kylehickinson ?

Reply:
I believe using  `UIUserInterfaceIdiom` might be good, cause the `DisplayMode` is only used in `updateLayoutForMode` method and it will be sufficient. However it is actually not big deal to use this enum and it might actually makes things less complicated while the updateLayoutMethod is called.